mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping

```diff
@@ -0,0 +1,48 @@
+CLASS net/minecraft/class_5455 net/minecraft/util/registry/DynamicRegistryManager
```

Comment:
Who do I have to mug to get a class javadoc over here? 
I have a hunch what this is for, but not really a goooood idea.

Reply:
Maybe don't be rude about it, because a javadoc isn't a requirement to get a PR merged... a good javadoc would look something like:
```suggestion
CLASS net/minecraft/class_5455 net/minecraft/util/registry/DynamicRegistryManager
	COMMENT Stores the registries that are not hardcoded. These registries may be changed for example via a datapack, and some may be synced with the client on login. For the vanilla values of these registries, see {@linkplain BuiltinRegistries}.
```

Reply:
Oh, sorry. I didn't intend to be rude, just snarky :<

Reply:
Time to review again. This time docs added.